elastic-job-lite-spring/src/main/resources/META-INF/namespace/job.xsd

```diff
@@ -38,7 +38,7 @@
                 <xsd:attribute name="monitor-port" type="xsd:string" default="-1"/>
                 <xsd:attribute name="max-time-diff-seconds" type="xsd:string" default="-1"/>
                 <xsd:attribute name="failover" type="xsd:string" default="false"/>
-                <xsd:attribute name="reconcile-interval-minutes" type="xsd:int" default="10"/>
+                <xsd:attribute name="reconcile-interval-minutes" type="xsd:string" default="10"/>
```

Comment:
Please use other pull request to change it. Let's make this pull request do upgrade quartz only.